lesson__2/lesson_1.js

```diff
@@ -0,0 +1,18 @@
+function isCapitalized(str) {
+  const firstChar = str.split('')[0];
+  const isAChar = new RegExp("^[a-zA-Z]+$");
```

Comment:
Тяжеловато читать имена переменных с артиклями. К тому же, char-character = символ. Лучше:
```suggestion
  const letterRegExp = new RegExp("^[a-zA-Z]+$");
```